Dockerfile

```diff
@@ -12,10 +12,17 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -q \
 
 ENV HOME /home/$USER
 
-WORKDIR $HOME/source
+WORKDIR $HOME
 
-COPY osmaxx-py $HOME/source
+# Little trick to avoid rebuilding this image over and over again.
```

Comment:
oh, so it isn't obvious... probably missing it, because I didn't know, what to write as an explanation; should probably just remove it, since it doesn't add anything useful.